Mathlib/Logic/Basic.lean

```diff
@@ -964,6 +966,41 @@ end congr
 
 end ite
 
+/-! ### Declarations about membership -/
```

Comment:
For ease of review, could you split out a PR that moves and golfs just this section?